db/migrate/20210820121752_add_magic_link_tokens_to_users.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class AddMagicLinkTokensToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :magic_link_token, :string, unique: true
```

Comment:
I don't think the 'unique' on here has done anything in the schema. We need a unique index for this.